courses/free-courses-id.md

```diff
@@ -405,6 +410,7 @@
 
 ### Rust
 
+* [Belajar bahasa pemrograman Rust!](https://www.youtube.com/watch?v=1gRIAVwDlb4&pp=ygUMYmVsYWphciBydXN0) - Ngooding
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes